### ISSUE TITLE
step-2 training

### ISSUE THREAD

**Issue by ziwenjie** [WeixiaoGao/PSSNet#2]
Hi, Doctor Gao:
I am very interest in your work, the step-1 has over-segmented the urban meshes with a very high accuracy. However, I use the released code step-2 to train the sum-helsinki dataset, the classification results is very bad. I tried many leanring-rate, and the results is bad too.
![微信图片_20230613122130](https://github.com/tudelft3d/PSSNet/assets/49607231/90457fe0-358d-4915-9aa9-53e1ed0bb801)

Best regard!

**Comment by WeixiaoGao** [comment]
Hi, thanks for your interest. If you keep the default parameters, you should be able to get similar results to those in the paper. So I don't think the learning rate is the problem. The problem is most likely in your input data. Can you check that the input data you got from Step 1 is correct and in the correct folder? Please refer to the closed [issue](https://github.com/tudelft3d/PSSNet/issues/1#issuecomment-1499629946).

**Comment by ziwenjie** [comment]
Thanks for your reply. The data is put in the right folder.
![IMG_20230613_145211.jpg](https://github.com/tudelft3d/PSSNet/assets/49607231/835da65e-ee40-4044-8ebb-9e09db5309b6)

![IMG_20230613_145153.jpg](https://github.com/tudelft3d/PSSNet/assets/49607231/622bd413-becc-48fe-918b-80c169835c68)
